x-pack/plugins/security_solution/public/common/components/toolbar/bulk_actions/helpers.ts

```diff
@@ -45,3 +45,17 @@ export const createInitialTagsState = (existingTags: string[][], defaultTags: st
     })
     .sort(checkedSortCallback);
 };
+
+export const getUpdateAlertsQuery = (eventIds: Readonly<string[]>) => {
+  return {
+    query: {
+      bool: {
+        filter: {
+          terms: {
```

Comment:
Well spotted, but this function will get deleted soon. 🙏 

I moved this function to a different file because I deleted most usages of it. Only one is left (recently introduced), but I believe it will get deleted soon because we can also use the `_bulk` API there.

Reply:
Do you want to mark it as deprecated to avoid devs using it?